benchmarks/src/main/kotlin/BenchmarkUtils.kt

```diff
@@ -0,0 +1,14 @@
+/*
+* Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+*/
+
+import java.util.concurrent.ThreadLocalRandom
+
+fun doGeomDistrWork(work: Int) {
```

Comment:
Have you verified that this function actually behaves like a geometric distribution when it is stressed on JVM?

Reply:
I've tested this function on different means, and created histograms of execution times. You can see the code here https://gist.github.com/fmixing/badbfc029e154486c8766ab9cc70a2a6
![Geom distr benchmark](https://user-images.githubusercontent.com/16418723/70476165-36f4c580-1ae7-11ea-8224-db8b28d76bf3.jpg)